raiden/raiden_service.py

```diff
@@ -358,7 +358,7 @@ def set_node_network_state(self, node_address, network_state):
     def start_health_check_for(self, node_address):
         self.protocol.start_health_check(node_address)
 
-    def poll_blockchain_events(self, current_block=None):  # pylint: disable=unused-argument
+    def poll_blockchain_events(self, block_number=None):  # pylint: disable=unused-argument
```

Comment:
What is the second argument event doing there? It was not used before and neither is it used now.

Reply:
This function is used as a callback for the alarm task, and all callbacks are called with the current block number as the default argument.

This argument is removed in the PR #1512